### PR TITLE
added multiple schema handling capbilities to reflect

### PR DESCRIPTION
**Pull request by tinnaing347**
closes
* #28 
* #27

### CODE REVIEW

**Comment by tinnaing347** [comment]
@bsnacks000 `mypy` is failing for now...if we use a list of schema instead of `iterable`, it may be easier to take care of `mypy` screaming at us.

**Comment by tinnaing347** [comment]
@bsnacks000 NOTE: `db.reflect` will reflects for

* if `sa.Metadata` is provided with a schema; i.e `metadata = sa.Metadata(schema="myschema")`
* if a list of schemas is provided: `schema=["myschema1", "myschema2"]`
* if a single schema is provided: `schema=myprecious`

In any case, `reflect` will reflects for `schema=None`; maybe twice in the case with `metadata = sa,Metadata(schema="myschema")`; not quite sure..

File: aiodal/dal.py [hunk 1]
```diff
@@ -66,28 +68,47 @@ async def reflect(
             resolve_fks (bool, optional): see `sa.MetaData.reflect`. Defaults to True.
 
         """
+        relect_metadata_first = True if metadata and metadata.schema else False
+
         if not metadata:
             metadata = sa.MetaData()
 
+        ls_schema = [None]  # for tables without schema name
+        # XXX probably should just limit to a list instead of iterable..mypy does not like it
```
Comment (bsnacks000) [inline aiodal/dal.py:77]: make it so then... no prob

File: tests/conftest.py [hunk 2]
```diff
@@ -23,5 +29,67 @@ class TestingEnum(Enum):
...
+        await db.reflect(async_engine, schema="testschema1")
+        yield db
+    except Exception as err:
+        logger.exception(err)
+    finally:
+        await async_engine.dispose()
+
+
+@pytest.fixture(scope="module")
+async def multischema_transaction(
```
Comment (bsnacks000) [inline tests/conftest.py:55]: :+1: I think this is ok... we can also extend the internal pytest fixtures to support this by extracting out the `schema` parameter

**Review by bsnacks000 — APPROVED** [review]
Just that one small change... make it a `List` if it will shut up mypy

**Pull request merged**